packages/server/postgres/queries/insertMeetingTemplate.ts

```diff
@@ -0,0 +1,18 @@
+import getRethink from '../../database/rethinkDriver'
+import MeetingTemplate from '../../database/types/MeetingTemplate'
+import getPg from '../getPg'
+
+const insertMeetingTemplate = async (meetingTemplate: MeetingTemplate) => {
+  const r = await getRethink()
+  const pg = getPg()
+  const {id, name, teamId, orgId, parentTemplateId, type} = meetingTemplate
+  await Promise.allSettled([
+    r.table('MeetingTemplate').insert(meetingTemplate).run(),
+    pg.query(
+      `INSERT INTO "MeetingTemplate" (id, name, "teamId", "orgId", "parentTemplateId", type) VALUES ($1, $2, $3, $4, $5, $6)`,
```

Comment:
-1 Both entries would differ in `createdAt` and `updatedAt` date. Also in theory `meetingTemplate` could contain a non-default `scope`, `lastUsedAt`, `isStarter` and `isFree` field.

Reply:
> RE: `createdAt, updatedAt`

yep, timestamps are gonna be off by <5ms. When we check for equality in phase 2 we'll have to account for that clock shift, it shouldn't be a problem.

> RE extra fields

You're absolutely correct! We currently don't allow setting those fields during creation & I didn't want to add any extra functionality during this refactor, but can see that it might act as a footgun if I don't. Happy to do it either way as it's pretty minimally invasive, just let me know what you think is best.

Reply:
-1 There is at least one place where we also have `scope` and `lastUsedAt`
https://github.com/ParabolInc/parabol/blob/ed7aae85321542057e0069946dc38b89a883e5ca/packages/server/graphql/mutations/updateTemplateScope.ts#L81-L88

Reply:
added those 2 + isFree, isStarter. future proof!